scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/vqfn.yaml

```diff
@@ -337,6 +337,48 @@ VQFN-24-1EP_4x4mm_P0.5mm:
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
 
+VQFN-28-1EP_4x4mm_P0.45mm_EP2.4x2.4mm:
+  device_type: 'VQFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'http://ww1.microchip.com/downloads/en/DeviceDoc/Atmel-9505-AT42-QTouch-BSW-AT42QT1060_Datasheet.pdf'
+  ipc_class: 'qfn'
+  #ipc_density: 'least'
+  body_size_x:
+    minimum: 3.95
+    maximum: 4.05
+  body_size_y:
+    minimum: 3.95
+    maximum: 4.05
+  body_height:
+    minimum: 0.8
+    maximum: 1
+  lead_width:
+    minimum: 0.17
+    maximum: 0.27
+  lead_len:
+    minimum: 0.35
+    maximum: 0.45
+
+  EP_size_x:
+    nominal: 2.4
+    tolerance: 0.05
+  EP_size_y:
+    nominal: 2.4
+    tolerance: 0.05
+  EP_num_paste_pads: [2, 3]
+
+  thermal_vias:
+    count: [3,3]
+    drill: 0.3
+    paste_via_clearance: 0.1
+    paste_avoid_via: False
```

Comment:
Increase the paste coverage for the thermal vias option as you have paste overlapping vias. (i would suggest EP_paste_coverage: 0.7)

Reply:
Changed. Is there any rule for setting the past coverage?

Reply:
Not really to be honest. I read a few application notes and discovered that there is quite a bit of range suggested (something between 0.5 and 0.8 seems to be optimal if there is no loss of solder. Depending a bit on the process used. So i normally use 0.65 as it is in the middle of that range.)

For footprints with vias covered by solder past i simply increase it by a bit. (In the past to 0.75 but that seems a bit excessive now so i suggest 0.7 to be used. I really have no particular reason for this suggestion to be honest. It is more of a guess than anything else.)